integration/kubernetes/helm-chart/monitor/templates/prometheus/service.yaml

```diff
@@ -0,0 +1,34 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+{{ if .Values.prometheus.enabled -}}
+{{- $name := include "monitor.name" . }}
+{{- $fullName := include "monitor.fullname" . }}
+{{- $chart := include "monitor.chart" . }}
+kind: Service
+apiVersion: v1
+metadata:
+  labels:
+    name: {{ $fullName }}-prometheus
+    app: {{ $name }}
+    chart: {{ $chart }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  name: {{ $fullName }}-prometheus
+spec:
+  type: NodePort
+  ports:
+    - port: 8081
```

Comment:
does this need to be a configuration?

Reply:
I will remove this hardcode, thanks for pointing this!